dataRetriever/mock/requestersContainerStub.go

```diff
@@ -0,0 +1,77 @@
+package mock
+
+import (
+	"github.com/multiversx/mx-chain-go/dataRetriever"
+)
+
+// RequestersContainerStub -
+type RequestersContainerStub struct {
```

Comment:
why not use the one from `testscommon/dataRetriever` ? `common_test.go` is part of `dataRetriever_test` package so there should be no cyclic dependency

Reply:
done